packages/artifact_proxy/config.yaml

```diff
@@ -0,0 +1,23 @@
+engine_mappings:
+  79f4c5321a581f580a9bda01ec372cbf4a53aa53:
+    flutter_engine_revision: 9aa7816315095c86410527932918c718cb35e7d6
+    shorebird_storage_bucket: download.shorebird.dev
+    shorebird_artifact_overrides:
```

Comment:
It almost feels like our workflow should involve recording per-revision and then filtering that recording to generate these?